.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "3.7"
+  
+services:
+  - postgresql 
+  
+install:
+  - pip install -r requirements.txt
+
+before_script:
+  - psql -c 'create database covid;' -U postgres
```

Comment:
@azaria-dedmon , there is no need to create the `covid db`, we only need to create the `covidtest db`.

Reply:
Okay. When I try to run the tests without the `covid` db and just `covidtest`, I receive an error and when I view the error from Travis it states, 

"**psycopg2.OperationalError: FATAL:  database "covid" does not exist
The above exception was the direct cause of the following exception:**"

However when I add, `covid` db to the "before_script", everything passes.

Reply:
@azaria-dedmon let me check it out.